include/deal.II/numerics/vector_tools_evaluate.h

```diff
@@ -134,6 +137,12 @@ namespace VectorTools
    *   cache so that the above function has to be called again. See also the
    *   example above.
    *
+   * @note If a point cannot be found, the result for these points will
+   *   be undefined (most probably 0). If you want to be sure that all
+   *   points receive a valid result,
+   *   RemotePointEvaluation::call all_points_found() before
+   *   calling this function.
```

Comment:
This content also needs to be part of the documentation of `RemotePointEvaluation`, probably `reinit()` (?). Maybe users don't want to use `RemotePointEvaluation` in combination with `VectorTools::point_values()`. Then, where do they find this delicate information?

Reply:
One more aspect: This comment reads like the point-search is done here, but hasn't it already been done before by RPE?